run.ts

```diff
@@ -1,23 +1,28 @@
-export type ShopifyFunction<Input extends {}, Output extends {}> = (
+export type userFunction<Input extends {}, Output extends {}> = (
```

Comment:
Any particular reason why then name of this type was updated?

Reply:
I think since it was `ShopifyFunction`, it was colliding with the same-named interface or global.

https://github.com/Shopify/shopify-function-javascript/blob/385a0a5dbb56fb47d2bc6c946b8f3aebd91d964b/run.ts#L5-L12

Reply:
Right, I was mostly asking because types are usually capitalized, `UserFunction`. Mind capitalizing this type?